signingscript/task.py

```diff
@@ -95,6 +95,27 @@ def _get_scope_prefix(context, sub_namespace):
     return '{}{}:'.format(prefix, sub_namespace)
 
 
+# get_autograph_retry_config {{{1
+def get_autograph_retry_config(context):
+    """Retrieve the autograph retry config from config.json or the default config.
+
+    Args:
+        context (Context): the signing context
+
+    Returns:
+        dict: with args to http://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.retry
+
+    """
+    retry_config = dict(
+        total=5,
+        backoff_factor=0.1,
+        method_whitelist=['POST'],
+        status_forcelist=[500, 502, 503, 504]
+    )
+    retry_config.update(context.config.get('autograph_retries', {}))
```

Comment:
Nit: I'm not what's the expected content of the dict. I didn't add a test for it. Could you tell me how it should look like @g-k ?

Reply:
Hm, I might have obsoleted that function in https://github.com/mozilla-releng/signingscript/pull/53/commits/b5d262365fc60c31c0eb6109a7612a88f36dcfdc . It's also possible I haven't removed all calls to it...

Reply:
@JohanLorenzo `get_autograph_retry_config` is dead code now that @escapewindow switched us to `scriptworker.utils.retry_async`.

Thanks for backporting the fixes!

Reply:
Okay. I removed this function, then!